build.gradle

```diff
@@ -98,8 +98,8 @@ android {
     useLibrary 'org.apache.http.legacy'
 
     defaultConfig {
-        versionCode 661
-        versionName "1.41.0"
+        versionCode 662
```

Comment:
we're getting dangerously close to `versionCode 666`

Reply:
oh!!! :sweat: